### PR TITLE
Javascript local storage - Tiffany Brown

### PR DESCRIPTION
**Pull request by tiffanydbrown**
Screenshot 2023-11-10 at 12 05 52 PM

### CODE REVIEW

File: javascript-local-storage/data.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* exported todos */
+
+let todos = [];
+
+const previousTodosJSON = localStorage.getItem('javascript-local-storage');
+todos = JSON.parse(previousTodosJSON);
```
Comment (shawnkost) [inline javascript-local-storage/data.js:6]: This should be inside an `if` statement making sure `previousTodosJSON` does not equal `null`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Tiffany!

**Comment by tiffanydbrown** [comment]
Screenshot 2023-11-13 at 8 58 47 AM

**Comment by tiffanydbrown** [comment]
Screenshot 2023-11-13 at 9 47 33 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**